frontend/src/components/regions/UsersAndGroups/Presentation.js

```diff
@@ -144,6 +144,7 @@ const UserCard = ({ email, user }) => {
         className={classes.title}
         label="View"
       />
+        <div>&nbsp;</div>
```

Comment:
Have you considered using styling? We usually handle spacing inside useStyles. 
Why would we do this one differently?